### PR TITLE
feat: add log level configuration to ssl communication

### PR DESCRIPTION
**Pull request by paulozulato**
Fixes https://emqx.atlassian.net/browse/EMQX-9781

### <samp>🤖 Generated by Copilot at 7717678</samp>

This pull request adds a new schema option `log.ssl.level` to configure the log level of SSL communication in the `emqx` application. It also updates the documentation and the translation files to reflect the new option.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] ~Added tests for the changes~
- [ ] ~Changed lines covered in coverage report~
- [x] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` files
- [x] For internal contributor: there is a jira ticket to track this change
- [ ] ~If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up~
- [ ] ~Schema changes are backward compatible~

## Checklist for CI (.github/workflows) changes

- [ ] ~If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)~
- [ ] ~Change log has been added to `changes/` dir for user-facing artifacts update~

### CODE REVIEW

File: apps/emqx/src/emqx_schema.erl [hunk 1]
```diff
@@ -2285,6 +2285,16 @@ common_ssl_opts_schema(Defaults) ->
...
                 }
             )},
+        {"log_level",
+            sc(
+                hoconsc:enum([
+                    emergency, alert, critical, error, warning, notice, info, debug, none, all
+                ]),
+                #{
+                    default => notice,
+                    desc => ?DESC(common_ssl_opts_schema_log_level)
```
Comment (zhongwencool) [inline apps/emqx/src/emqx_schema.erl:2295]: Maybe mark log_level as `importance => ?IMPORTANCE_LOW` is better ？ 
The number of users who need to modify this parameter should be small.

Comment (paulozulato) [reply apps/emqx/src/emqx_schema.erl:2295]: OK

**Review by lafirest — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

**Comment by yanzhiemq** [comment]
_

**Comment by yanzhiemq** [comment]
_

**Pull request merged**